opencensus-migration/src/main/java/SpanCache.java

```diff
@@ -0,0 +1,70 @@
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import io.opencensus.implcore.trace.RecordEventsSpanImpl;
+import io.opencensus.trace.Span;
+import java.util.concurrent.TimeUnit;
+
+public class SpanCache {
+
+  private static final SpanCache SPAN_CACHE = new SpanCache();
+
+  private static final int MAXIMUM_CACHE_SIZE = 10000;
+  private static final int CACHE_EXPIRE_TIME = 10;
```

Comment:
Maybe I should make this configurable by the applications, WDYT?

Reply:
Making it configurable is problematic since we are instantiating SpanCache statically :(